src/node-lib/getServerConfig.test.ts

```diff
@@ -19,6 +19,30 @@ describe("getServerConfig()", () => {
 
     expect(getServerConfig("disableIsr")).toEqual(false);
   });
+  it("Should return the default value if a required env is set to undefined", async () => {
+    process.env.SANITY_DATASET_TAG = undefined;
+    const { default: getServerConfig } = await import(
+      "../node-lib/getServerConfig"
+    );
+
+    expect(getServerConfig("sanityDatasetTag")).toEqual("default");
+  });
+  it("Should return the default value if a required env is set to an empty string", async () => {
+    process.env.SANITY_DATASET_TAG = "";
+    const { default: getServerConfig } = await import(
+      "../node-lib/getServerConfig"
+    );
+
+    expect(getServerConfig("sanityDatasetTag")).toEqual("default");
+  });
```

Comment:
nit-pick not for this specific PR: feels a tad funky having the real keys and values in here instead of some explicit testing values

_but_ it doesn't really matter and these lines are just an extension of how it already is

Reply:
when we break this out into a reusable module we should take care of that

Reply:
marking resolved so I can merge, but it's a bit annoying, I'd rather this comment remained visible